Templates/Kotlin/InputWrapper.stencil

```diff
@@ -3,23 +3,20 @@ import javax.annotation.Generated
 
 @Generated("com.shopify.syrup")
 class InputWrapper<T>() {
-    private var value: T? = null
-    private var defined = false
```

Comment:
Now that we removed the `isDefined` function, should we rename the property to `isDefined`?

Reply:
updated